index.bs

```diff
@@ -8546,6 +8564,23 @@ entails.
 
 <h4 id="LegacyWindowAlias" extended-attribute lt="LegacyWindowAlias">[LegacyWindowAlias]</h4>
 
+<div class="advisement">
+
+    [{{LegacyWindowAlias}}] [=extended attributes=] are an undesirable feature.
+    They exist only so that legacy Web platform features can be specified.
+    They should not be used in specifications
+    unless required to specify the behavior of legacy APIs.
+
+    <small class="non-normative">
+        The [{{LegacyWindowAlias}}] [=extended attribute=] appears on the following [=interfaces=]:
```

Comment:
> It just feels a bit backward...

I agree. I'm fighting my own internal purist here to just do the pragmatic thing which is to list these things and be done with it.